cmd/kubeadm/app/preflight/checks.go

```diff
@@ -653,7 +653,7 @@ func (swc SwapCheck) Check() (warnings, errorList []error) {
 	}
 
 	if len(buf) > 1 {
-		return []error{errors.New("swap is enabled; production deployments should disable swap unless testing the NodeSwap feature gate of the kubelet")}, nil
+		return []error{errors.New("swap is supported for cgroup v2 only; the NodeSwap feature gate of the kubelet is beta but disabled by default, and better keep it off")}, nil
```

Comment:
```suggestion
		return []error{errors.New("swap is supported for cgroup v2 only; the NodeSwap feature gate of the kubelet is beta but disabled by default")}, nil

i think we can just say this part and let the user decide

Reply:
/hold

Reply:
pls let this pr hold for an while to wait for @pacoxu to address this comment.

Reply:
Thanks @chendave. Updated.